gatorgrouper_django/gatorgrouper/migrations/0001_initial.py

```diff
@@ -0,0 +1,68 @@
+# Generated by Django 2.1.7 on 2019-02-15 21:15
```

Comment:
This file is still not removed from the actual PR, however -- you need to unstage/remove it and push.

Reply:
Deleted File